src/codeu/chat/server/Server.java

```diff
@@ -96,16 +96,56 @@ public void onMessage(InputStream in, OutputStream out) throws IOException {
                 final String name = Serializers.STRING.read(in);
                 final User user = controller.newUser(name);
 
+<<<<<<< HEAD
```

Comment:
Looks like maybe you had merge conflicts that were not fully resolved. Could you please fix them?